logic/tournamentSettingsLogic.js

```diff
@@ -0,0 +1,59 @@
+'use strict'
+const MongoClient = require('mongodb').MongoClient
+const {MHUB_NODES, publishMsg} = require('../Utils/mhubConnection')
+
+const MONGU_URI = process.env.MONGO_URI
+const SETTING_COLLECTION_NAME = 'settings'
+
+function setDefaultSettings () {
+  const defaultSettings = {
+    'tournamentLevel': 'practice',
+    'tournamentTitle': 'World Festival 2018'
+  }
+  MongoClient.connect(MONGU_URI).then(connection => {
+    connection.db().collection(SETTING_COLLECTION_NAME).findOne().then(response => {
+      if (!response) {
+        return connection.db().collection(SETTING_COLLECTION_NAME).insert(defaultSettings)
+      }
+    })
+  })
+}
+
+function getSetting (settingName) {
+  return MongoClient.connect(MONGU_URI).then(connection => {
+    return connection.db().collection(SETTING_COLLECTION_NAME)
+      .findOne({})
+      .then(data => {
+        if (!data) {
+          return data
+        }
+
+        return data[settingName]
+      })
+  })
+}
+
+function updateSetting (settingName, value) {
+  return MongoClient.connect(MONGU_URI).then(connection => {
+    const setDocument = {}
+    setDocument[settingName] = value
+    return connection.db().collection(SETTING_COLLECTION_NAME)
+      .findOneAndUpdate({}, {$set: setDocument})
+      .then(dbResponse => {
+        if (dbResponse.ok == 1) {
+
+          if(settingName == 'tournamentLevel'){
```

Comment:
You should publish the title as well using `title:updated` and givving its value

Reply:
could it be the full setting name?

Reply:
Sure, but change the `tournamentLevel` to `tournamentStage` because it's called stage everywhere else

Reply:
Yes, I Agree on that